### PR TITLE
ARROW-9580: [JS][Doc] Fix syntax error in example code

### PR DESCRIPTION
**Pull request by domoritz**

### CODE REVIEW

**Comment by emkornfield** [comment]
@TheNeuralBit @trxcllnt could one of you review?

**Review by TheNeuralBit — APPROVED** [review]
Thanks @domoritz! LGTM

Did you mean to tag a different Jira? ARROW-8580 has to do with pyarrow and it's closed

**Comment by domoritz** [comment]
Ups. Yes, I fixed the reference.

File: js/README.md [hunk 1]
```diff
@@ -122,7 +122,7 @@ const rainfall = Table.new(
 ```js
 import { Table } from "apache-arrow";
 
-const table = await Table.from(fetch(("/simple.arrow")));
+const table = await Table.from(fetch("/simple.arrow"));
```
Comment (t829702) [inline js/README.md:125]: where can I download a copy of this `/simple.arrow` file?

**Comment by t829702** [comment]
the https://arrow.apache.org/docs/js/ documentation is still much less, incomplete if comparing to other implementations, Especially can someone explain how to write arrow files? before I can start play with `apache-arrow` JS package, where is a place can I download some sample arrow files? how can I convert my existing dataset (some csv, json, and avro files) to arrow format? 
https://arrow.apache.org/docs/js/classes/recordbatchwriter.html (where are howto using RecordBatchWriter, RecordBatchFileWriter, & RecordBatchStreamWriter)

Wonder is this [Implementation Status](https://arrow.apache.org/docs/status.html) kept current? Look at other releases have 1.0.0 and 1.0.1 but on NPM the apache-arrow is still left at 0.17

File: js/README.md (hunk 1 above)
Comment (domoritz) [reply js/README.md:125]: Not sure but that's beyond the scope of this fix.

**Pull request closed without merging**